### DISCUSSION TITLE
Function to open app in full screen mode

### DISCUSSION THREAD

**Discussion by berma83** [Budibase/budibase#2324] [Help]
Hello,

Is there any way to open the application in full screen mode? 

I would like to open the application in full screen on mobile phones.

**Comment by joebudi** [comment]
I'm not sure what you mean. Could you elaborate or send a screenshot?

**Reply by berma83** [reply]
https://youtu.be/4nUnlYeqOUg

**Reply by mjashanks** [reply]
I think you are asking for a Progressive Web App (PWA). It is not possible to do this at the moment, although something we have often discussed. It's not going to happen in the next few months, but we would like to do this.

**Comment by GuilhermeHirsch** [comment]
Hi, @berma83! How are you? No browser navigation bar?

**Reply by berma83** [reply]
Sorry. Yes, without navigation bar.

**Reply by GuilhermeHirsch** [reply]
In web app style. I'm also very interested in this.

**Reply by mjashanks** [reply]
If you only want to remove the Budibase nav bar, you can do this using `Layouts`. You can create your own layouts, but get 2 by default:

![image](https://user-images.githubusercontent.com/3524181/128996577-3d55e892-71f6-4bf3-8adf-4357d8365918.png)

In a layout settings, you can choose the navigation type = `Top`, `Left` or `None`.

You can then click on a screen, and choose which layout you want. In your case, `Empty Layout` will be without a navigation.
